packages/mui/src/date-time-pickers/wrapper.tsx

```diff
@@ -29,19 +34,38 @@ type PickerProps<
 type Props<P extends PickerTypes, In, Out> = SubsetPartial<
   PickerProps<P, In, Out>,
   'renderInput'
->;
+> &
+  (P extends typeof TimePicker ? unknown : { multiLocale?: boolean });
```

Comment:
put `{ multiLocale?: undefined }` instead of `unknown`

Reply:
Ok

Reply:
No this will not meet my need. I don't want multiLocale prop to be visible for the TimePicker component.